api/src/main/java/org/apache/cloudstack/api/command/user/network/CreateNetworkCmd.java

```diff
@@ -16,13 +16,13 @@
 // under the License.
 package org.apache.cloudstack.api.command.user.network;
 
+import org.apache.cloudstack.api.BaseCmd;
 import org.apache.log4j.Logger;
 
 import org.apache.cloudstack.acl.RoleType;
 import org.apache.cloudstack.api.APICommand;
 import org.apache.cloudstack.api.ApiConstants;
 import org.apache.cloudstack.api.ApiErrorCode;
-import org.apache.cloudstack.api.BaseCmd;
```

Comment:
Import reshuffle does not look sortable. You can fix your intellij settings or get preset settings (mine are here https://github.com/rhtyd/dotfiles/blob/master/intellij/settings.jar)